services/records_to_agol.py

```diff
@@ -29,6 +30,32 @@ def format_filter_date(date_from_args):
     return "1970-01-01" if not date_from_args else arrow.get(date_from_args).isoformat()
 
 
+def resilient_layer_request(func, args, max_retries=MAX_RETRIES):
```

Comment:
we could definitely use function decorators here. i suppose i'm just being lazy, but also decorators with argument become a bit messy. this feels like a clean enough solution.